documentation/modules/auxiliary/scanner/http/http_hsts.md

```diff
@@ -0,0 +1,66 @@
+## Vulnerable Application
+
+### Description
+
+This module checks to see whether or not the scanned systems return the HSTS header to enforce HSTS.
+
+## Verification Steps
+
+1. Do: ```use auxiliary/scanner/http/http_hsts```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```set RPORT [PORT]```
+4. Do: ```run```
+
+## Options
+
+  **Proxies**
+
+  Any proxies to be used, in the format format type:host:port[,type:host:port][...]. Default is `` or none.
+
+  **RHOSTS**
+
+  The target hosts to be scanned. Can be a single host, a range CIDR identifier, hosts file with syntax `file:<path>`. Default is ``.
+
+  **RPORT**
+
+  Port to scan on the targets.  Default is `443`.
+
+  **SSL**
+
+  Should SSL be used when connecting? Default is `TRUE`.
+
+  **THREADS**
+  Number of threads to use to complete the scan. Default is `1`.
+
+  **VHOST**
+  Virtual host of the HTTP servers being scanned. Default is ``.
+
+## Scenarios
+
+**Running the scanner**
```

Comment:
This should be an H3 that says what you are scanning (ie ### Apache 2.2 on Ubuntu 18.04)

Reply:
Sorry to be clear should **Running the scanner** should be changed to a H3 with the target that was scanned?

Reply:
You can remove that, it should look more like https://github.com/rapid7/metasploit-framework/pull/12423/files#diff-d2659524190bc3d103650ebcd869e3e0R101

Reply:
Pushing an update which should fix this in a sec.